cmd/celestia-appd/cmd/root.go

```diff
@@ -165,6 +167,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig encoding.Config) {
 		queryCommand(),
 		txCommand(),
 		keybase,
+		qgbcmd.VerifyCmd(),
```

Comment:
something we might want to think about is adding this command to the query command, since this is a query

Reply:
that's a good idea. The reason I didn't do it  is that the query commands  somehow are  consistent: they use the RPC/GRPC  endpoints to query things from state, like everything needed is gotten from core/app. However, in this case, we're also connecting to an EVM chain and interacting with a  smart contract.

I am okey with either tbh, let me know if I should change